services/catalog/Makefile

```diff
@@ -17,7 +17,7 @@ requirements: ## compiles pip requirements (.in -> .txt)
 .PHONY: install-dev install-prod install-ci
 install-dev install-prod install-ci: requirements _check_venv_active ## install app in development/production or CI mode
 	# installing in $(subst install-,,$@) mode
-	pip-sync requirements/$(subst install-,,$@).txt
+	pip-sync --quiet requirements/$(subst install-,,$@).txt
```

Comment:
I just had an issue installing the dependencies, I would like to see the result of the command without the `--quiet` flag